lib/tasks/cron.rake

```diff
@@ -7,5 +7,6 @@ namespace :cron do
     ImportGiasDataJob.schedule
     ValidationRetryJob.schedule
     SchoolAnalyticsJob.schedule
+    CreateNewFakeSandboxDataJob.schedule if Rails.env.sandbox?
```

Comment:
Looks like you have a guard (a different guard) at the beginning of `perform`

Reply:
Yeah, cause I am paranoid that a dev will accidentally run this job where it shouldn't be ran 😂